src/main/webapp/script.js

```diff
@@ -0,0 +1,114 @@
+
+import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
+
+// global objects
+let scene;
+let camera;
+let renderer;
+let clock;
+let controls;
+const rotationData = new Map();
+const points = [];
+
+/**
+ * Intializes the scene and its child objects
+ */
+function init() {
+  scene = new THREE.Scene();
+  camera = new THREE.PerspectiveCamera(
+      75, window.innerWidth / window.innerHeight, .1, 1000);
```

Comment:
maybe add comments to describe each parameter meaning. e,g,
camera = new THREE.PerspectiveCamera(
      /\*fov =\*/75, /\*aspect =\*/window.innerWidth / window.innerHeight, /\*near =\*/.1, /\*far =\*/1000)